ビルドエラー

#285 のマージ後からビルドエラー

```
remote: Counting objects: 23, done.
remote: Compressing objects: 100% (23/23), done.
remote: Total 23 (delta 11), reused 0 (delta 0), pack-reused 0
Unpacking objects: 100% (23/23), done.
From github.com:readcrx-2/read.crx-2
e/Notification.coffee src/core/ReadState.coffee src/core/ReplaceStrTxt.coffee src/core/Thread.coffee src/core/WriteHistory.coffee src/core/thread_search.coffee src/core/util.coffee | node_modules/.bin/coffee -cbs > /var/folders/x5/0pj_cm0
15yj21ph2p3_qc5540000gn/T/d20170506-17304-1c13r7j/_coffee.js
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/Bookmark.ChromeBookmarkEntryList.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/Bookmark.CompatibilityLayer.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/Bookmark.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/HTTP.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/URL.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/core/Util.ts
cat src/core/Util.js src/core/URL.js src/core/Bookmark.js src/core/HTTP.js src/core/Bookmark.CompatibilityLayer.js src/core/Bookmark.ChromeBookmarkEntryList.js > /var/folders/x5/0pj_cm015yj21ph2p3_qc5540000gn/T/d20170506-17304-1c13r7j/ts.js
rm src/core/Util.js src/core/URL.js src/core/Bookmark.js src/core/HTTP.js src/core/Bookmark.CompatibilityLayer.js src/core/Bookmark.ChromeBookmarkEntryList.js
cat /var/folders/x5/0pj_cm015yj21ph2p3_qc5540000gn/T/d20170506-17304-1c13r7j/ts.js /var/folders/x5/0pj_cm015yj21ph2p3_qc5540000gn/T/d20170506-17304-1c13r7j/_coffee.js > debug/app_core.js
cat src/cs_addlink.coffee | node_modules/.bin/coffee -cbs > debug/cs_addlink.js
mkdir -p debug/img
convert src/image/svg/read.crx.svg          \( -clone 0 -resize 16x16 \)          \( -clone 0 -resize 32x32 \)          -delete 0          debug/img/favicon.ico
convert -background transparent -resize 96x96 -extent 128x128-16-16 src/image/svg/read.crx.svg debug/img/read.crx_128x128.png
convert -background transparent -resize 48x48 src/image/svg/read.crx.svg debug/img/read.crx_48x48.png
convert -background transparent -resize 16x16 src/image/svg/read.crx.svg debug/img/read.crx_16x16.png
convert -background transparent -resize 16x16 src/image/svg/close.svg debug/img/close_16x16.webp
convert -background transparent -resize 1x1 src/image/svg/dummy.svg debug/img/dummy_1x1.webp
convert -background transparent -resize 100x100 src/image/svg/loading.svg debug/img/loading.webp
convert -background transparent -fill '#3a5' -opaque '#333' -resize 12x12 src/image/svg/lock.svg debug/img/lock_12x12_3a5.webp
convert -background transparent -fill '#333' -opaque '#333' -rotate '90' -resize 19x19 src/image/svg/arrow.svg debug/img/arrow_19x19_333_r90.webp
convert -background transparent -fill '#333' -opaque '#333' -rotate '-90' -resize 19x19 src/image/svg/arrow.svg debug/img/arrow_19x19_333_r-90.webp
convert -background transparent -fill '#777' -opaque '#333' -resize 19x19 src/image/svg/search2.svg debug/img/search2_19x19_777.webp
convert -background transparent -fill '#333' -opaque '#333' -resize 19x19 src/image/svg/star.svg debug/img/star_19x19_333.webp
convert -background transparent -fill '#007fff' -opaque '#333' -resize 19x19 src/image/svg/star.svg debug/img/star_19x19_007fff.webp
convert -background transparent -fill '#333' -opaque '#333' -resize 19x19 src/image/svg/reload.svg debug/img/reload_19x19_333.webp
convert -background transparent -fill '#333' -opaque '#333' -resize 19x19 src/image/svg/pencil.svg debug/img/pencil_19x19_333.webp
convert -background transparent -fill '#333' -opaque '#333' -resize 19x19 src/image/svg/menu.svg debug/img/menu_19x19_333.webp
convert -background transparent -fill '#182' -opaque '#333' -resize 19x19 src/image/svg/lock.svg debug/img/lock_19x19_182.webp
convert -background transparent -fill '#333' -opaque '#333' -resize 19x19 src/image/svg/unlock.svg debug/img/unlock_19x19_333.webp
convert -background transparent -fill '#ddd' -opaque '#333' -rotate '90' -resize 19x19 src/image/svg/arrow.svg debug/img/arrow_19x19_ddd_r90.webp
convert -background transparent -fill '#ddd' -opaque '#333' -rotate '-90' -resize 19x19 src/image/svg/arrow.svg debug/img/arrow_19x19_ddd_r-90.webp
convert -background transparent -fill '#aaa' -opaque '#333' -resize 19x19 src/image/svg/search2.svg debug/img/search2_19x19_aaa.webp
convert -background transparent -fill '#ddd' -opaque '#333' -resize 19x19 src/image/svg/star.svg debug/img/star_19x19_ddd.webp
convert -background transparent -fill '#f93' -opaque '#333' -resize 19x19 src/image/svg/star.svg debug/img/star_19x19_f93.webp
convert -background transparent -fill '#ddd' -opaque '#333' -resize 19x19 src/image/svg/reload.svg debug/img/reload_19x19_ddd.webp
convert -background transparent -fill '#ddd' -opaque '#333' -resize 19x19 src/image/svg/pencil.svg debug/img/pencil_19x19_ddd.webp
convert -background transparent -fill '#ddd' -opaque '#333' -resize 19x19 src/image/svg/menu.svg debug/img/menu_19x19_ddd.webp
convert -background transparent -fill '#3a5' -opaque '#333' -resize 19x19 src/image/svg/lock.svg debug/img/lock_19x19_3a5.webp
convert -background transparent -fill '#ddd' -opaque '#333' -resize 19x19 src/image/svg/unlock.svg debug/img/unlock_19x19_ddd.webp
bundle exec scss -E UTF-8 -t compressed src/ui/ui.scss debug/ui.css
cat src/ui/Animate.coffee src/ui/PopupView.coffee src/ui/TableSorter.coffee src/ui/ThreadContent.coffee src/ui/ThreadList.coffee src/ui/contextmenu.coffee src/ui/dialog.coffee src/ui/table_search.coffee | node_modules/.bin/coffee -cbs > /var/folders/x5/0pj_cm015yj21ph2p3_qc5540000gn/T/d20170506-17304-5jpeuv/_coffee.js
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/ui/Accordion.ts
node_modules/.bin/tsc --target es2015 --lib dom,es2015,es2016 --skipLibCheck --noUnusedLocals --alwaysStrict src/ui/LazyLoad.ts
src/ui/LazyLoad.ts(183,7): error TS2322: Type 'Element[]' is not assignable to type 'HTMLMediaElement[]'.
  Type 'Element' is not assignable to type 'HTMLMediaElement'.
    Type 'Element' is not assignable to type 'HTMLVideoElement'.
      Property 'height' is missing in type 'Element'.
rake aborted!
Command failed with status (2): [node_modules/.bin/tsc --target es2015 --li...]
/Users/eru/Hack/read.crx-2/Rakefile:102:in `block in typescript'
/Users/eru/Hack/read.crx-2/Rakefile:101:in `typescript'
/Users/eru/Hack/read.crx-2/Rakefile:126:in `block (2 levels) in file_ct'
/Users/eru/Hack/read.crx-2/Rakefile:118:in `block in file_ct'
Tasks: TOP => default => ui:build => debug/ui.js
(See full trace by running task with --trace)
```

--- a/src/ui/LazyLoad.ts
+++ b/src/ui/LazyLoad.ts
@@ -180,7 +180,7 @@ namespace UI {
     }
 
     scan (): void {
-      this.medias = Array.from(this.container.querySelectorAll("img[data-src], audio[data-src], video[data-src]"));
+      this.medias = <HTMLMediaElement[]>Array.from(this.container.querySelectorAll("img[data-src], audio[data-src], video[data-src]"));
       if (this.medias.length > 0) {
         this.update();
         this.watch();

